var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -333,6 +339,17 @@ def common_config_args(self):
         else:
             config_args.append('-no-freetype')
 
+        if self.spec.variants['xcb'].value == 'spack':
+            config_args.extend([
+                '-system-xcb',
+                '-I{0}/xcb'.format(self.spec['xcb-util'].prefix.include)
+            ])
+
+        elif self.spec.variants['xcb'].value == 'qt':
+            config_args.append('-qt-xcb')
```

Comment:
What does this option do? Does it build against a vendored copy of xcb? We generally avoid that and only build against Spack packages to avoid different versions in the DAG.

Reply:
Yes. That's why it's a variant. There is a similar variant for `freetype`. Yes,  dependency on libxcb and `xcb-*` should conditional (again, similar to `freetype`)

Reply:
I would avoid ever using a system version or vendored copy of a dependency and only rely on a Spack installation. If it can be built without xcb, then I would make it a boolean variant instead of a multi-valued variant.